### ISSUE TITLE
[JOSS Review] Performance claims

### ISSUE THREAD

**Issue by arokem** [bmcfee/resampy#41]
How would I confirm the performance claims of the software? An example that demonstrates the speedup benefits would be a useful addition to the documentation, IMHO.

**Comment by bmcfee** [comment]
Great idea.  I have this implemented in notebooks that could be readily pushed into the documentation.

Here's a quick example from the scipy lightning talk:

```python
import numpy as np
import scipy
import resampy

x = np.random.randn(1, 400000)
sr_in, sr_out = 22050, 16000

%timeit resampy.resample(x, sr_in, sr_out, axis=-1)

%timeit scipy.signal.resample(x, int(x.shape[1] * sr_out / sr_in), axis=-1)
```

**Comment by arokem** [comment]
I am much more impressed now. Feel free to close this issue.

**Comment by arokem** [comment]
By the way, is there a video of this lightning talk somewhere? You might want to add that to the docs as well.

**Comment by bmcfee** [comment]
Benchmark example is now up in the #42 docs.

**Comment by bmcfee** [comment]
Fixed by merging #42
